src/Symfony/Component/Semaphore/Store/scripts/redis_save.lua

```diff
@@ -0,0 +1,43 @@
+local key = KEYS[1]
+local weightKey = key .. ":weight"
+local timeKey = key .. ":time"
+local identifier = ARGV[1]
+local now = tonumber(ARGV[2])
```

Comment:
Given time is provided by php script, what if the application use several servers that are not time synchronized?

Reply:
This is a huge issue indeed. All server should be synchonise.
Note: I can not use the `TIME` command from redis because it is not [deterministic](https://redis.io/commands/eval#replicating-commands-instead-of-scripts)